pkg/util/fs/fs.go

```diff
@@ -281,8 +281,8 @@ func (h *fs) RemoveDirectory(dir string) error {
 	// TODO: Remove this workaround when we switch to go 1.7 -- os.RemoveAll should
 	// be fixed for Windows in that release.  https://github.com/golang/go/issues/9606
 	if runtime.GOOS == "windows" {
-		cmd := exec.Command("cmd.exe", "/c", fmt.Sprintf("rd /s /q %s", dir))
-		output, err := cmd.Output()
+		command := exec.Command("command.exe", "/c", fmt.Sprintf("rd /s /q %s", dir))
```

Comment:
watch your global search and replace.... unless you really intended to rename the executable here.

Reply:
fixed, thanks